addons/web_editor/static/src/js/wysiwyg/plugin/keyboard.js

```diff
@@ -900,6 +947,33 @@ var KeyboardPlugin = AbstractPlugin.extend({
         }
         return range;
     },
+    /**
+     * Select all the contents of the current unbreakable ancestor.
+     */
+    _selectAll: function () {
+        var range = this.context.invoke('editor.createRange');
+        var unbreakable = dom.ancestor(range.sc, this.options.isUnbreakableNode);
+        var $contents = $(unbreakable).contents();
+        var pointA = {
```

Comment:
Why not using `HelperPlugin.makePoint()` ?

Reply:
Good point (pun intended).